packages/kit/src/provider/Container/InAppNotification/index.tsx

```
@@ -34,6 +57,178 @@ const InAppNotification = () => {
+        swapApprovingTransaction: undefined,
+      }));
+    }
+  }, [
+    intl,
+    setInAppNotificationAtom,
+    swapApprovingTransaction?.resetApproveIsMax,
+    swapApprovingTransaction?.resetApproveValue,
+    swapApprovingTransaction?.status,
+    swapApprovingTransaction?.txId,
+    approvingSuccessAction,
+  ]);
```

ESLint

[error] React Hook useEffect has a missing dependency: 'swapApprovingTransaction'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)